CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## v0.5.0-alpha
+* Deprecated metadata json format.
+* Removed `TransactionAsyncMetadataResponse` model.
+* Added support to metadata protobuf format.
+* `GetTransactionMetadata` returns protobuf metadata.
+
```

Comment:
Oh, I didn't realize that there were version numbers already in the CHANGELOG. ... 🤔 

@vilterp  and I created two new versions while you were out - versions 0.2.0 and 0.2.1:
https://github.com/RelationalAI/rai-sdk-go/tags

But maybe those should have already been 0.4.0?

How are you currently indicating the version numbers other than in this changelog file? It seems that the Go package manager finds version numbers through github tags, which is why we added those. But maybe we were missing some other existing mechanism?

Reply:
I'm not sure what to do about the version numbers we already tagged... 😅 Maybe it's fine to delete them and set the tags back to match up with version numbers in this file? Or maybe we just leave them and then skip straight to `v0.5.0-alpha`?

Reply:
I was planning to add a workflow to index this package to `pkg.go.dev` but didn't implement that :sweat_smile:, happy to hear that the Go package manager finds version numbers also through github tags :smile:

Reply:
So how should we tag a new release, then? Maybe we can discuss tomorrow? Thanks helmi!